### PR TITLE
CSI-1737 advance version to 1.3.0

### PR DESCRIPTION
**Pull request by oriyarde**
similar PR in driver: https://github.com/IBM/ibm-block-csi-driver/pull/222

### CODE REVIEW

File: version/version.go [hunk 1]
```diff
@@ -17,6 +17,6 @@
 package version
 
 var (
-	Version       = "1.2.0"
-	DriverVersion = "1.2.0"
+	Version       = "1.3.0"
+	DriverVersion = "1.3.0"
```
Comment (ArbelNathan) [inline version/version.go:21]: what the diff?

Comment (oriyarde) [reply version/version.go:21]: what do you mean?

Comment (ArbelNathan) [reply version/version.go:21]: version vs DriverVersion

Comment (oriyarde) [reply version/version.go:21]: [your guess is as good as mine](https://github.com/IBM/ibm-block-csi-operator/pull/76/files#r438116211).
`DriverVersion` sounds pretty straightforward, and I see `Version` usage [here](https://github.com/IBM/ibm-block-csi-operator/blob/aa140208f60135ad1e7819bb85ce53bfe63d8f8d/pkg/internal/ibmblockcsi/ibmblockcsi.go#L72)

Comment (ArbelNathan) [reply version/version.go:21]: so Driver is controller an Node? like in the github?

Comment (ArbelNathan) [reply version/version.go:21]: ok, sounds right, if we will have patches it will be handy to separate it..

**Review by ArbelNathan — APPROVED** [review]
LGTM

**Review by barakda — APPROVED** [review]

**Pull request merged**